composeApp/src/commonMain/kotlin/pl/edu/uw/juwenalia/data/source/StaticMapData.kt

```diff
@@ -0,0 +1,107 @@
+package pl.edu.uw.juwenalia.data.source
+
+import io.github.dellisd.spatialk.geojson.Position
+import pl.edu.uw.juwenalia.data.model.MapPoint
+import pl.edu.uw.juwenalia.data.model.MapPointCategory
+
+object StaticMapData {
+    const val FESTIVAL_PLACE_LATITUDE = 52.220692
+    const val FESTIVAL_PLACE_LONGITUDE = 21.043575
+    const val FESTIVAL_PLACE_DEFAULT_ZOOM = 16.2
+    const val MAP_POINT_FOCUS_ZOOM = 17.0
+
+    val mapPoints =
+        listOf(
+            MapPoint(
```

Comment:
te dane to powinny się chyba wczytywać z jsona (albo w jakiś inny sposób), a nie być zahardkodowane